Jenkinsfile

```diff
@@ -59,27 +44,59 @@ pipeline {
         }
       }
     }
-    stage('Dockerize') {
+    stage('Docker Build') {
       when {
         not { expression { isPr } }
       }
       steps {
         script {
           lastStage = env.STAGE_NAME
           if (params.RELEASE_BUILD) {
-            env.REGISTRY_PUSH = (params.RELEASE_DRY_RUN) ? 'none' : 'all'
-            env.DOCKER_TAG = dockerize.docker_build_specify_tag_and_push(params.RELEASE_NAME, sbtbuild.getDockerPath(), sbtbuild.getDockerArtifact(), env.REGISTRY_PUSH)
+            env.DOCKER_TAG = dockerize.dockerBuildWithSpecificTag(tag: params.RELEASE_NAME, path: sbtbuild.getDockerPath(), artifacts: [sbtbuild.getDockerArtifact()])
```

Comment:
For future reference, separating more than 3 arguments onto separate lines makes reading diffs easier.

Reply:
Done